test/server.spec.js

```diff
@@ -2,16 +2,24 @@ const { createTestClient } = require('apollo-server-testing'),
   { ApolloServer } = require('apollo-server'),
   schema = require('../app/graphql');
 
-const { query: _query, mutate } = createTestClient(
-  new ApolloServer({
-    schema,
-    formatError: err => ({
-      message: err.message,
-      statusCode: err.extensions.code
-    })
+ApolloServer.prototype.setContext = function setContext(newContext) {
+  this.context = newContext;
+};
+
+ApolloServer.prototype.resetContext = function resetContext() {
```

Comment:
do we really need this?

Reply:
It is necessary to imitate the function of context on the real server